src/Lykke.Service.Limitations/Modules/ServiceModule.cs

```diff
@@ -17,34 +21,22 @@ namespace Lykke.Service.Limitations.Modules
 {
     public class ServiceModule : Module
     {
-        private readonly AppSettings _appSettings;
-        private readonly IReloadingManager<LimitationsSettings> _settingsManager;
-        private readonly ILog _log;
-
-        public ServiceModule(
-            AppSettings appSettings,
-            IReloadingManager<LimitationsSettings> settingsManager,
-            ILog log)
+        private readonly IReloadingManager<AppSettings> _appSettings;
+        
+        public ServiceModule(IReloadingManager<AppSettings> appSettings)
         {
-            _appSettings = appSettings;
-            _settingsManager = settingsManager;
-            _log = log;
+            _appSettings = appSettings;            
         }
 
         protected override void Load(ContainerBuilder builder)
         {
-            builder.RegisterInstance(_log).As<ILog>().SingleInstance();
-
-            var settings = _appSettings.LimitationsSettings;
-
-            builder.Register(context => ConnectionMultiplexer.Connect(settings.RedisConfiguration))
+            builder.Register(context => ConnectionMultiplexer.Connect(_appSettings.CurrentValue.LimitationsSettings.RedisConfiguration))
```

Comment:
I think it's better to reuse _appSettings.CurrentValue instead of calling each time.